src/main/java/com/hms_networks/americas/sc/extensions/system/time/SCTimeUtils.java

```diff
@@ -14,8 +18,29 @@
   /** Key for accessing the time zone name item in the Flexy's system control block. */
   private static final String SCB_TIME_ZONE_KEY = "Timezone";
 
+  /** The name of the Java time zone representing GMT/UTC. */
+  private static final String GMT_TIME_ZONE_NAME = "GMT";
+
+  /** The identifier used for the GMT/UTC time zone, which has no offset. */
+  private static final String GMT_TIME_ZONE_IDENTIFIER = "Z";
+
+  /** The simple date format pattern used for representing ISO 8601 date and time strings. */
+  private static final String SIMPLE_DATE_FORMAT_PATTERN_ISO_8601 = "yyyy-MM-dd'T'HH:mm:ss.SSS";
+
+  /** The simple date format representing ISO 8601 format with the time zone as UTC. */
+  private static SimpleDateFormat iso8601UtcTimeFormat = null;
+
+  /**
+   * The simple date format representing ISO 8601 format with the local time zone offset appended
+   * (or Z if the time zone is GMT/UTC).
+   */
+  private static SimpleDateFormat iso8601LocalTimeFormat = null;
+
+  /** The identifier for the local time zone. (i.e. -05:00, +02:00, Z, etc.) */
```

Comment:
Specs call this Time Zone Designator (TDZ)

Reply:
updated references